### ISSUE TITLE
Release notes for 1.11.0

### ISSUE THREAD

**Issue by qianmoQ** [EdurtIO/dbm#94]
**Release time**

`2022-02-22`

**Components involved**

- ClickHouse

**Check list**

#### Enhancement
  ---

  - [x] Nullable decorated column type is supported
  - [x] Build tables support optional parameters
  - [x] Support version update check
  - [x] Support to clear partition data by filtering criteria
  - [x] Supports data migration

  #### Docs
  ---

  - [x] Support For Chinese documents

  #### Optimize
  ----

  - [x] Optimize database deletion UI

  #### Bug
  ---

  - [x] Fixed inability to build Log related engine tables
  - [x] Fix slow query list cannot display DDL
  - [x] Fix README.md file logo cannot be displayed

**Comment by joint-song** [comment]
I found issues as followed in this release:
1. Tables and databases information bar inside `Management` didn't refresh automatically while after deletion some tables or databases;
2. There's a typo: `Tools`.`Migrte`(Maybe `Migrate` or Using `Migrte` as intended?)
3. I tested `Support to clear partition data by filtering criteria` functionality against `system`.`query_log`, but after load the partitions, there were too many duplicated button showed in window. That looked wired.
